tensorflow/contrib/crf/python/kernel_tests/crf_test.py

```diff
@@ -199,6 +200,87 @@ def testViterbiDecode(self):
       self.assertEqual(actual_max_sequence,
                        expected_max_sequence[:sequence_lengths])
 
+  def testCrfDecode(self):
+
+    def np_crf_decode(potentials, transition_params, seq_lens):
```

Comment:
I think it's fine. If `viterbi_decode` has been tested (and is correct) and tensor version has same result with it, we know tensor version is correct too.